.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
It would really be great if we can avoid publishing personalized setting even if they are turned into comments